apps/mobile/src/queries/defi/bitflow.query.ts

```diff
@@ -0,0 +1,21 @@
+import { toFetchState } from '@/components/loading';
```

Comment:
Thanks for adding these examples 👍   maybe they should go into the Web App though as we aren't using these on mobile and I'm not sure if we will have portfolio / positions there? (We should probably show these somewhere on mobile) 

I wonder should we move back towards having more queries shared from `packages/query` that accept network / currency as params? That could help us create a more unified presentation layer for the UI that can do more data manipulation in a shared way